pr.ts

```diff
@@ -26,16 +26,20 @@ export const changelog = wrap("Require changelog entries on PRs with code change
   const getContentParams = { path: "", owner: pr.head.user.login, repo: pr.head.repo.name }
   const rootContents: any = await danger.github.api.repos.getContent(getContentParams)
 
-  const hasChangelog = rootContents.data.find(file => changelogs.includes(file.name))
-  if (hasChangelog) {
+  const hasChangelog = rootContents.data.find((file: any) => changelogs.includes(file.name))
+  const markedTrivial = pr.title.match("#trivial")
+  if (hasChangelog && !markedTrivial) {
     const files = [...danger.git.modified_files, ...danger.git.created_files]
 
     // Look for Swift files that aren't in a unit test directory.
-    const hasCodeChanges = files.find(file => file.match(/.*\.swift/) && !file.match(/(test|spec)/i))
+    const hasCodeChanges = files.find((file: any) => file.match(/.*\.swift/) && !file.match(/(test|spec)/i))
```

Comment:
do we need the explicit `any` type here? Seemed to have worked before without it. Same for the `hasChangelog` definition on L29

Reply:
Good question! This isn't necessary but was raising a TypeScript warning that `file` was being implicitly typed as `any` and that we should add a type specifier. It's very similar to some of the warnings you get in Swift.

Reply:
Oh OK cool :) 

In Swift you don't get warning for inferred types, but I don't have enough (read- any) experience with TS unfortunately, so I take your word on that 😄

Reply:
They don't come up very often, but I've seen warnings about implicit casts to `Any` in Swift (when working with Objective-C APIs, mostly).